pkg/ebpf/ingress_node_firewall_loader.go

```diff
@@ -25,12 +26,15 @@ const (
 // IngNodeFwController structure is the object hold controls for starting
 // ingress node firewall resource
 type IngNodeFwController struct {
+	mu sync.Mutex
 	// eBPF objs to create/update eBPF maps
 	objs BpfObjects
 	// eBPF interfaces attachment objects
-	links []link.Link
+	links map[string]link.Link
 	// eBPF pingPath
 	pinPath string
+	// refCnt how many interfces use this BpfObjects
+	refCnt int
```

Comment:
Instead of using refCnt, would it make sense to:
~~~
delete(ifaceName, infc.links)
if len(ifnc.links) <= 0 {
...
}
~~~

Reply:
agree will remove the counter